lib/cc/analyzer/container.rb

```diff
@@ -124,6 +124,15 @@ def container_data(duration: nil, status: nil)
           @stderr_io.string
         )
       end
+
+      def reap_running_container
+        container_id = `docker ps --quiet --filter name=#{@name}`.chomp
```

Comment:
Unfortunately not true for me. Can you test, maybe it was a version
difference?

On Tue, Oct 20, 2015, 18:33 Bryan Helmkamp <notifications@github.com> wrote:

> In lib/cc/analyzer/container.rb
> <https://github.com/codeclimate/codeclimate/pull/154#discussion_r42563788>
> :
>
> > @@ -124,6 +124,15 @@ def container_data(duration: nil, status: nil)
> >            @stderr_io.string
> >          )
> >        end
> > +
> > +      def reap_running_container
> > +        container_id = `docker ps --quiet --filter name=#{@name}`.chomp
>
> docker kill (and all docker commands0 accept a container name as a
> substitute for the container ID. Therefore, I think the name lookup is
> unnessary.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codeclimate/codeclimate/pull/154/files#r42563788>.
>